app/main.ipynb

```diff
@@ -403,23 +514,66 @@
     "result"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T19:25:17.607239Z",
+     "start_time": "2024-11-07T19:25:17.598474Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: total: 0 ns\n",
+      "Wall time: 0 ns\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 43,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 43
   },
   {
    "cell_type": "code",
-   "execution_count": 24,
-   "outputs": [],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "def unknown_signature(vector1, vector2):\n",
+    "    vector1 = [1, 2, 3]\n",
+    "    vector2 = [4, 5, 6]  \n",
+    "    a = np.array(vector1)\n",
+    "    b = np.array(vector2)\n",
+    "\n",
+    "    result = sum(a * b)\n",
+    "    print(result)\n",
+    "%time unknown_signature(vector1, vector2)"
```

Comment:
The function `unknown_signature` should not redefine `vector1` and `vector2` inside the function. Use the parameters passed to the function instead to maintain flexibility and reusability.